### ISSUE TITLE
Increased fecal bile acid excretion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7152]
**Preferred term label:**
Increased fecal bile acid excretion

**Synonyms**
Increased fecal bile acid 

**Definition (free text, please give PubMed ID)**
Elevated amount of bile acids in the feces. 
Comment:Bile acids are synthesized from cholesterol in the liver and have major roles in the absorption of lipids in the small intestine. After their involvement in micelle formation in the small intestine, they are usually effectively reabsorbed in the terminal ileum, and efficiently (over 95%) reabsorbed by the liver from the portal venous blood, and resecreted.

**Parent term (use hpo.jax.org/app)**
Abnormal stool composition HP:0031685

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613291 
PMID: 9109432   1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
